alpine-dependencies-dev.list

```diff
@@ -0,0 +1,5 @@
+# Install packages to images docker, "common" is declared into dependencies-extra of alpine6-node-base.
+add="$common autoconf automake bash bzip2 libffi-dev libpng libpng-dev nasm nodejs=4.3.0-r0 ruby ruby-dev"
+
+# Delete packages to images docker, "common" is declared into dependencies-extra of alpine6-node-base.
+del="$common autoconf automake bzip2 libffi-dev libpng libpng-dev ruby ruby-dev"
```

Comment:
This file is almost duplicated. As we've talked before we can execute always the dependencies-extra and later the dependencies-dev to remove duplication.